e2e/testcases/apiservice_test.go

```diff
@@ -37,6 +37,12 @@ func TestCreateAPIServiceAndEndpointInTheSameCommit(t *testing.T) {
 		if t.Failed() {
 			nt.PodLogs(adapterNamespace, adapterName, "pod-custom-metrics-stackdriver-adapter", true)
 		}
+		if !nt.MultiRepo {
+			nt.MustKubectl("delete", "-f", "../testdata/apiservice/apiservice.yaml", "--ignore-not-found")
+			nt.MustKubectl("delete", "-f", "../testdata/apiservice/rbac.yaml", "--ignore-not-found")
+			nt.MustKubectl("delete", "-f", "../testdata/apiservice/namespace-custom-metrics.yaml", "--ignore-not-found")
+			nt.MustKubectl("delete", "-f", "../testdata/apiservice/namespace.yaml", "--ignore-not-found")
+		}
```

Comment:
Why not remove them from the repository and let Config Sync prune them?

Reply:
Don't know why I added them, they are getting removed from git through Config Sync.